Add doc https ssl (#2657)
GeotrekCE/Geotrek-admin: docs/configuration.rst

    sudo dpkg-reconfigure geotrek-admin

Activate SSL / HTTPS
--------------------

To activate https, you need firstly to change custom.py and add :

::

    SESSION_COOKIE_SECURE = True
    CSRF_COOKIE_SECURE = True

After this, edit `nginx.conf.in` to add your certificate.

If you generate it with letsencrypt :
You can use certbot to add the certificate in your configuration.
But you will have to move the configuration automatically added into `nginx.conf`, to the file `nginx.conf.in`

> in `/opt/geotrek-admin/var/conf/` directory

Reviewer: je pense qu'il faut expliquer pourquoi on doit bouger la conf

You can also create groups, etc. See PostgreSQL documentation.

> Activate SSL / HTTPS

Reviewer 2: I would put this section just after nginx configuration

--------------------

> If you want to activate https, you need firstly to change custom.py and add :

Reviewer 2: I would say "To activate HTTPS… " instead of "If you want to activate…" because it is mandatory in many case
```suggestion
To activate https, you need firstly to change custom.py and add :
```

::

    SESSION_COOKIE_SECURE = True
    CSRF_COOKIE_SECURE = True

> After this, modify the configuration of nginx.conf.in, add your certificate.

Reviewer 2: Markdown code
```suggestion
After this, edit `nginx.conf.in` to add your certificate.
```

If you generate it with letsencrypt :
You can use certbot to add the certificate in your configuration.

> But, you need to move the configuration added automatically to nginx.conf in the file nginx.conf.in

Reviewer 2: Markdown & english grammar
```suggestion
But you will have to move the configuration automatically added into `nginx.conf`, to the file `nginx.conf.in`
```

> inside the directory /opt/geotrek-admin/var/conf/

Reviewer 2: Markdown & english grammar
```suggestion
in `/opt/geotrek-admin/var/conf/` directory
```
